app/ai/voice/agents/breeze_buddy/mcp/__init__.py

```
@@ -14,26 +17,336 @@
+
+    try:
+        if ui_hint.trigger == ToolUiTrigger.SKIP_UI:
+            parsed["_ui_skip"] = True
+        else:
+            if ui_hint.instructions:
+                parsed["_ui_instructions"] = ui_hint.instructions
+            if ui_hint.examples:
+                parsed["_ui_examples"] = [
+                    ex.model_dump(exclude_none=True) for ex in ui_hint.examples
+                ]
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)